android/cli/commands/_build.js

```diff
@@ -3406,6 +3409,101 @@ AndroidBuilder.prototype.generateI18N = async function generateI18N() {
 	}
 };
 
+AndroidBuilder.prototype.generateSemanticColors = async function generateSemanticColors() {
+	this.logger.info(__('Generating semantic colors resources'));
+	const _t = this;
+	const xmlFileName = 'semantic.colors.xml';
+	const valuesDirPath = path.join(this.buildAppMainResDir, 'values');
+	const valuesNightDirPath = path.join(this.buildAppMainResDir, 'values-night');
```

Comment:
Don't create a "values.xml" file. We risk file collision with the app developer's own "res" files.
They need to be prefixed with "ti_" or "ti-". (C-style convention.)

So, we should name the files "ti_semantic_values.xml" and "ti_semantic_values_night.xml".
Or "ti-semantic-values.xml" and "ti-semantic-values-night.xml".

Note: Drawable "res" files can't have dash '-' chars (they're illegal), but XML "res" files can.

Reply:
@jquick-axway These aren't creating a values.xml file, they're creating:
- `build/app/src/main/res/values/semantic.colors.xml`
- `build/app/src/main/res/values-night/semantic.colors.xml`

Is the suggestion we should instead name them:
- `build/app/src/main/res/ti_semantic_colors.xml`
- `build/app/src/main/res/ti_semantic_colors_night.xml`
?